roles/tendrl-storage-node/tasks/rsyslog.yml

```diff
@@ -0,0 +1,6 @@
+---
+
+- name: ensure rsyslog is running
+  service:
+    name=rsyslogd
+    state=started
```

Comment:
Here we are going a bit deeper than originally intended. While this makes sense, shouldn't we make sure that the service is enabled as well? It's on the same level as making sure that the rsyslog is running.

Reply:
i will add enabled=yes ?